dll/win32/shell32/CDefView.cpp

```diff
@@ -1734,11 +1734,12 @@ LRESULT CDefView::OnSize(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandled
 
     TRACE("%p width=%u height=%u\n", this, wWidth, wHeight);
 
+    // WM_SIZE can come before WM_CREATE
```

Comment:
Is it understood why and how this wm_size can come before the wm_create? And whether this happens on Windows as well?

Reply:
I don't know the reason.
Also Windows does so.

Reply:
@HBelusca there are more messages that come before wm_create, like wm_nccreate and wm_nccalcsize. I guess if an app does something non trivial there could trigger this wm_size. nice catch